cmd/builder/README.md

```diff
@@ -131,6 +131,16 @@ replaces:
   - github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.40.0
 ```
 
+The builder also allows setting the scheme to use as the default URI scheme via `resolver_settings.default_uri_scheme`:
+
+```yaml
+resolver_settings:
```

Comment:
```suggestion
conf_resolver_settings:
```

I think this needs some kind of indication that this is configuration resolver specific, not network resolvers

Reply:
Updated